lib/newPage.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
```

Comment:
@humphd Are you okay with this? You suggested just `<html>` tags, though I don't see the harm in this approach.